perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupsManagerEntryIntegrationTest.java

```diff
@@ -651,6 +651,26 @@ public void transitiveGroupMembershipCheck() throws Exception {
 
 	}
 
+	@Test
+	public void getGroupUnions() throws Exception {
```

Comment:
We have also tests: ``createGroupUnion()`` and ``removeGroupUnion()``. They for now check results only by members. Can you please add call ``groupsManagerBl.getGroupUnions()`` and check if they are ok by group relations too. Thank you. Also it would be nice, if they check ``membershipType`` and ``sourceGroupId`` to be the correct one. Not sure if we test it anywhere (I do in a one test for ``membershipType``, but group are related by hierarchy there, not by weak relation (inclusion)).